source/fundamentals/bson.txt

```diff
@@ -210,6 +210,30 @@ Driver will marshal structs using the following rules:
       - Includes an empty ``lastname`` field
       - Stores the ``Address`` field as a nested value
 
+BSON Options
+------------
+
+You can set ``BSONOptions`` to specify how
+the driver marshals and unmarshals BSON on your collection, database, or
+client.  
+
+The following example demonstrates how to set ``BSONOptions`` for 
+the ``movies`` collection:
+
+.. code-block:: go
+   :copyable: false
+
+   bsonOpts := &options.BSONOptions{
+      UseJSONStructTags: true,
+	}
+   coll := client.Database("sample_mflix").Collection("movies",
+   bsonOpts)
```

Comment:
S: I'm not a Go expert, but structuring the code like the following seems more readable:

```suggestion
   bsonOpts := &options.BSONOptions {
       UseJSONStructTags: true
   }
   coll := client.Database("sample_mflix")
       .Collection("movies", bsonOpts)
```
What do you think?

Reply:
Agree with everything, but comma on line 227 is needed